views/index.ejs

```diff
@@ -92,11 +92,30 @@
     <section class="slice clearfix">
         <div class=" clearfix">
             <div class="container" id="threeColumns">
-                <% if (openhabMajorVersion == '2') { %>
-                    You are using openHAB 2.x. <a href="https://home.myopenhab.org/start/index">Click here to access your openHAB's dashboard</a>
-                <% } else { %>
-                    You are using openHAB 1.x. To remotely access your openHAB's web interface go to <code>https://myopenhab.org/openhab.app?sitemap=yoursitemapname</code>
-                <% } %>
+                <%
+                    switch (openhabstatus) {
```

Comment:
Hmm, if so the OH cloud should set this to undefined, if I change the UUID to a new fake value (which was never ever connected so far). But it doesn't :)